user_docs/en/userGuide.t2t

```diff
@@ -2938,6 +2938,15 @@ Please see your display's documentation for descriptions of where these keys can
 | Move braille display to previous line | d1 |
 | Move braille display to next line | d3 |
 | Route to braille cell | routing |
+| shift+tab key | space+dot1+dot3 |
+| tab key | space+dot4+dot6 |
+| alt key | space+dot1+dot3+dot4 (space+m) |
+| escape key | space+dot1+dot5 (space+e) |
+| windows key | space+dot3+dot4 |
+| alt+tab key | space+dot2+dot3+dot4+dot5 (space+t) |

```

Comment:
Could the keys be wrapped in monospace formatting?

```suggestion
| ``alt+tab`` key | ``space+dot2+dot3+dot4+dot5`` (``space+t``) |
```

Reply:
It seems to me that many of these gesture tables don't markup the keys like that yet. I'm unclear as to whether it should be done for script names as well? E.g. "scroll forward"? 
Perhaps this could be done as a separate pr that goes through the user guide as a whole, as it will make the diff harder to read if I go through and doo the entire table here...

Reply:
The userGuide standards only suggest doing this for key commands https://github.com/nvaccess/nvda/blob/master/devDocs/userGuideStandards.md